pootle/apps/pootle_statistics/utils.py

```diff
@@ -83,4 +83,4 @@ def contributors(self):
         return OrderedDict(
             [(user["username"], user)
              for user
-             in qs.values("username", "contributions", "email")])
+             in qs.values("username", "full_name", "contributions", "email")])
```

Comment:
It is `display_name` what was used before. In order to replicate what `display_name` does we have to retrieve both `username` and `full_name`.